Created Enterprise security and compliance section (#2088)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

To view the history of your previous downloads, go to `edge://downloads`.  To clear your browsing data and delete your download history, go to `edge://settings/clearBrowserData`.  Deleting your download history from Microsoft Edge does not remove the files from your device.  Deleting downloaded files from your device does not remove the files from your download history.  When you use **InPrivate** browsing or **Guest** mode, the download history from the session is cleared when you close the **InPrivate** or **Guest** windows. The files remain saved on the device.

<!-- ====================================================================== -->

> ## Enterprise Security & Compliance

Reviewer: ```suggestion
## Enterprise Security and Compliance

```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand

Author: Working with the team to understand if this is a brand name. If not, I will make the '&' update and make change Security and Compliance to lowercase

> Microsoft Edge supports a variety of first party enterprise security & compliance features including, Windows Information Protection, Microsoft Defender Application Guard, Microsoft Endpoint DLP, and Microsoft Insider Risk management. These features become functional only on an Enterprise managed devices either by acquiring the right license for the feature or the administrator deploying specific policies on the device.

Reviewer: ```suggestion
Microsoft Edge supports a variety of first party enterprise security and compliance features including, Windows Information Protection, Microsoft Defender Application Guard, Microsoft Endpoint DLP, and Microsoft Insider Risk management. These features become functional only on an Enterprise managed devices either by acquiring the right license for the feature or the administrator deploying specific policies on the device.
```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand

> Microsoft Edge supports a variety of first party enterprise security and compliance features including, [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip), [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview), [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about), and [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview). These features become functional only on enterprise managed devices by acquiring the right license for the feature or the administrator deploying specific policies on the device.

Reviewer: ```suggestion
Microsoft Edge supports a variety of first-party enterprise security and compliance features, including:
* [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip)
* [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview)
* [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about)
* [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview) 

These features become functional only on enterprise-managed devices by acquiring the right license for the feature or by the administrator deploying specific policies on the device.
```

<!-- ====================================================================== -->
## Enterprise security and compliance

Microsoft Edge supports a variety of first-party enterprise security and compliance features, including:
* [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip)
* [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview)
* [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about)
* [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview) 

These features become functional only on enterprise-managed devices by acquiring the right license for the feature or by the administrator deploying specific policies on the device.

When these enterprise features are on, the browser sends auditing and diagnostic data to the administrator as per the feature’s requirement to function, manage or diagnose properly. This data collection is controlled by the enterprise administrator and end user is not able to opt-out.

> Please contact your enterprise administrator to understand your company’s policies for such data collection.

Reviewer: ```suggestion
Please contact your enterprise administrator to understand your company's policies for such data collection.
```

To view the history of your previous downloads, go to `edge://downloads`.  To clear your browsing data and delete your download history, go to `edge://settings/clearBrowserData`.  Deleting your download history from Microsoft Edge does not remove the files from your device.  Deleting downloaded files from your device does not remove the files from your download history.  When you use **InPrivate** browsing or **Guest** mode, the download history from the session is cleared when you close the **InPrivate** or **Guest** windows. The files remain saved on the device.

<!-- ====================================================================== -->

> ## Enterprise security and compliance

Reviewer: ```suggestion
## Enterprise security and compliance

```
blank line convention

> Microsoft Edge supports a variety of first party enterprise security and compliance features including, [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip), [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview), [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about?view=o365-worldwide), and [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview?view=o365-worldwide). These features become functional only on enterprise managed devices by acquiring the right license for the feature or the administrator deploying specific policies on the device.

Reviewer: ```suggestion

Microsoft Edge supports a variety of first-party enterprise security and compliance features, including [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip), [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview), [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about?view=o365-worldwide), and [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview?view=o365-worldwide). These features become functional only on enterprise-managed devices by acquiring the right license for the feature or by the administrator deploying specific policies on the device.
```
